templates/blog/newsletter-form.html

```diff
@@ -29,6 +29,13 @@ <h3 class="p-card--title p-heading--four">Select topics you&rsquo;re <br />inter
           </label>
         </li>
 
+        <li class="p-list__item u-no-margin--top u-clearfix">
+          <label>
```

Comment:
Why do we wrap these in label? Could these just use `for` and `id` like here: https://docs.vanillaframework.io/base/forms/#checkbox

Reply:
@anthonydillon I'll leave as it is for now, and can change it once the vertical spacing is reduced in Vanilla.